torch/testing/_internal/common_methods_invocations.py

```diff
@@ -68,18 +78,30 @@ def is_active(self, cls_name, test_name, device_type, dtype):
             (self.dtypes is None or dtype in self.dtypes)
         )
 
-
 class SkipInfo(DecorateInfo):
     """Describes which test, or type of tests, should be skipped when testing
        an operator. Any test that matches all provided arguments will be skipped.
-       The skip will only be checked if the active_if argument is True."""
+       The skip will only be checked if the active_if argument is True.
+       """
 
     def __init__(self, cls_name=None, test_name=None, *,
                  device_type=None, dtypes=None, active_if=True):
         super().__init__(decorators=skipIf(True, "Skipped!"), cls_name=cls_name,
                          test_name=test_name, device_type=device_type, dtypes=dtypes,
                          active_if=active_if)
 
+class FailureInfo(DecorateInfo):
```

Comment:
New class for asserting failures. This is better than skipping tests when the problem is something that needs to be fixed, so that when it is fixed we are reminded to remove the FailureInfo.